apps/nextjs-website/src/components/organisms/TabSwitcherProvider/TabSwitcherProvider.tsx

```diff
@@ -0,0 +1,92 @@
+import React, {
+  useState,
+  createContext,
+  useContext,
+  ReactNode,
+  SetStateAction,
+  Dispatch,
+} from 'react';
+import { Box, useTheme } from '@mui/material';
+
+type TabSwitcherContextType = {
+  setActiveTabID: Dispatch<SetStateAction<string>>;
+  activeTabID: string;
+};
+
+//the name of this context will be TabSwitcherContext
+const TabSwitcherContext = createContext<TabSwitcherContextType | null>(null);
+
+type TabProps = {
+  id: string;
+  children: ReactNode;
+};
+
+function Tab({ id, children }: TabProps) {
+  const context = useContext(TabSwitcherContext);
+  const { palette } = useTheme();
+  if (!context) {
+    console.error(
+      'TabSwitcherContext not found, did you forget to use TabSwitcherContextProvider?'
+    );
```

Comment:
Is this `console.error` needed?

Reply:
I think is useful to have a specific error during development. By default React does not show a comprehensive error when component's provider is missing.

Reply:
What about using an `Error` instead?